markdown/2017-10-23.md

```diff
@@ -0,0 +1,179 @@
+emerging trends at the cross over of business and technology. With cloud, big
+data and artificial intelligence already rewriting the rules of business and
+competition in every industry, it is essential that whatever your role, you
+understand the impact of technology on your business and on your career.
+
+With automation set to change the way we work in the coming years, Microsoft
+Future Decoded will provide insight into the skills and knowledge you will
+need in business and in technology to thrive in today's world of cloud, data,
+AI, Blockchain and virtual worlds. Plus, learn how to prepare for the seismic
+impact of tomorrow's computing, based on DNA and quantum computers.
+
+*Please note that this event is not organised by DoCSoc*
+
+# Microsoft Imagine
+
+Microsoft Imagine connects students with the tools, resources and experiences
+they need to elevate their skills for today's working world. Whether it's
+building a game, designing an app or launching a project, Microsoft Imagine
+helps students develop their ideas and boldly bring them to life.
+
+[Find out more about Microsoft Imagine](https://imagine.microsoft.com/en-us)
+
+---
+
+# AWS Workshoe
+
+Imperial College Advanced Hackspace (ICAH) are running an AWS workshop:
+
+- **When:** 13.00-17.00, 1st November 2017
+- **Where:** Royal School of Mines (RSM)
+- **Link:** [Eventbrite](https://www.eventbrite.co.uk/e/aws-101-introduction-to-amazon-web-services-tickets-38439665073)
+
+Interested in learning cloud technology and not sure where to start?
+Come join us in the upcoming AWS 101 workshop! This is a free introductory
+hands-on session which covers some useful services such as elastic computing
+(EC2) and Simple Storage Service (S3). You will also get an overview of what
+Amazon Web Service and find out what other services might be useful for you.
+
+This workshop is suitable for all and beginner friendly. Space limited,
+first-come-first-serve basis. Sign up soon!
+
+*Please note that this event is not organised by DoCSoc*
+
+# Academic Research Reading Groups
+
+We are excited to announce our first ever Reading Group event series! Based
+on member polls, the top fields of interest were: Security/Cryptography,
+Artificial Intelligence and Programming Languages.
+
+First and foremost, our own DoCSoc reading group in Programming Languages
+will be held in Huxley 145 on Tuesday, October 31 at 1-2pm. Cutting edge
+research papers published at leading Programming Languages conferences, such
+as OOPSLA, will be presented by their authors. You can find details about the
+speakers and the papers at <https://docsoc.co.uk/papers>. *This event is
+aimed at 3rd and 4th year students, however everyone interested in
+programming languages is welcome to attend.*
+
+As for Security/Cryptography and Artificial Intelligence, we have partnered
+with Ben Livshits and Marc Deisenroth to advertise their own reading groups.
+You can find the links on the DoCSoc website. *These events are aimed at
```

Comment:
just provide a link here.

Reply:
they likely change. The copy is by Rachel